api/tests/unit/segment_membership/test_unit_segment_membership_tasks.py

```
@@ -139,6 +139,25 @@ def test_seed_organisation_identities__insert_fails__logs_and_continues(
+    sql, rows_arg = cursor.executemany.call_args.args
+    assert sql == (
+        "INSERT INTO IDENTITIES "
+        "(environment_id, identifier, identity_key, traits, is_deleted) VALUES"
+    )
+    assert {row[0] for row in rows_arg} == {environment.api_key}
+    assert {row[1] for row in rows_arg} == {"a", "b"}
+    assert any(
+        e["event"] == "backfill.environment.completed" and e["rows__count"] == 2
+        for e in log.events
+    )
+    refresh_dispatch.delay.assert_called_once_with(args=(project.id,))
```

Ruff (0.15.21)

[error] Undefined name `refresh_dispatch`

(F821)